modules/documentation/introduction.md

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I guess this is also still a draft? If not it is a bit limited for self-learning.